src/common/utils/generateScheduleMatrix.ts

```diff
@@ -19,13 +19,14 @@ export const generateScheduleMatrix = <T extends Pair,>(weekSchedule: Schedule<T
     .map(() => new Array(DAYS.length).fill(null));
 
   const activePair: number = getActiveTimePoint();
-  const currentDay = moment().day();
+  const currentDay = dayjs().day();
 
   weekSchedule.forEach((schedule) => {
     const yIndex = DAYS.findIndex((item) => item === schedule.day);
 
     schedule.pairs.forEach((pair) => {
-      const xIndex = TIME_POINTS.findIndex((item) => item === pair.time);
+      const normalizedPairTime = createDateTime(pair.time);
+      const xIndex = TIME_POINTS.findIndex((item) => item === normalizedPairTime.format('HH:mm'));
```

Comment:
You could format `normalizedPairTime` before using it inside `.findIndex` function.

Reply:
Yes, good idea. Done